lib/ansible/modules/system/xfconf.py

```diff
@@ -136,13 +143,12 @@ def main():
         argument_spec=dict(
             channel=dict(required=True, type='str'),
             property=dict(required=True, type='str'),
-            value_type=dict(required=False,
-                            choices=['int', 'bool', 'float', 'string'],
-                            type='str'),
-            value=dict(required=False, default=None, type='str'),
+            value_type=dict(required=False, type='list'),
+            value=dict(required=False, type='list'),
```

Comment:
To address the comment https://github.com/ansible/ansible/issues/46308#issuecomment-463506618 I believe the `dict` for `value_type` and `value` should not specify a type, leave them free form
```suggestion
            value=dict(required=False),
```